packages/plugins/plugin-adjust/src/AdjustPlugin.tsx

```diff
@@ -16,8 +16,8 @@ export class AdjustPlugin extends DestinationPlugin {
   type = PluginType.destination;
   key = 'Adjust';
 
-  private adjust: AdjustConfig | undefined;
   private settings: SegmentAdjustSettings | null = null;
+  private registeredCallback: Boolean = false;
```

Comment:
Lets use `hasRegisteredCallback` to make it more aligned with the flags:

```suggestion
  private hasRegisteredCallback: Boolean = false;
```